lib/components/sidebar/sidebar.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Drawer as AntDrawer } from 'antd';
+import Container from './container';
+import styles from './sidebar.css';
+
+const Sidebar = React.memo(
+  ({ visibleDrawer, children, width, placement, onClose, closable, style }) => {
+    return (
+      <AntDrawer
+        {...{
+          className: styles.sidebarWrap,
+          visible: visibleDrawer,
+          closeIcon: null,
+          width,
+          placement,
+          onClose,
+          closable,
+          style: {
```

Comment:
Nitpick: Can we `useMemo` on the `style` prop value?

Reply:
I think yes. fixed.